plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -24,42 +19,31 @@ import { getAdminLink } from '@woocommerce/settings';
 import './attribute-field.scss';
 import { AddAttributeModal } from './add-attribute-modal';
 import { EditAttributeModal } from './edit-attribute-modal';
+import { EnhancedProductAttribute } from '~/products/hooks/use-product-attributes';
 import {
 	getAttributeKey,
 	reorderSortableProductAttributePositions,
 } from './utils';
-import { sift } from '../../../utils';
 import { AttributeEmptyState } from '../attribute-empty-state';
 import {
 	AddAttributeListItem,
 	AttributeListItem,
 } from '../attribute-list-item';
 
 type AttributeFieldProps = {
-	value: ProductAttribute[];
+	attributes: ProductAttribute[];
```

Comment:
Could we actually keep this as `value`? This will make things easier for consistency and re-usability to have any component depicted as a `field` to have a `value` and `onChange` callback.

Reply:
Nvm the above comment, I wasn't aware of the `Attributes` wrapper component that contains the `value` prop. We may want to rename this component to `Attributes` instead and the `Attributes` component to `AttributesField`. But that might be something we can do as part of the extensibility work.

Reply:
I think you're right on this with `value`.  I reverted back and also updated the component name to `AttributeControl` in https://github.com/woocommerce/woocommerce/pull/36354/commits/4e14bae129640340a32195ae04cbf1f6c63e6d14 to make it clear that this is a control component.

Reply:
Thanks for changing it back :) and also appreciate the renaming to `AttributeControl` that does make more sense.